platform/rct/hal/imu_stm32/imu_device_stm32.c

```diff
@@ -86,33 +86,27 @@ void imu_device_init()
 
 int imu_device_read(enum imu_channel channel)
 {
-    unsigned int ret = 0;
-
-    switch(channel) {
-    case IMU_CHANNEL_X:
-        ret = read_buf->accel.accel_x;
-        break;
-    case IMU_CHANNEL_Y:
-        ret = read_buf->accel.accel_y;
-        break;
-    case IMU_CHANNEL_Z:
-        ret = read_buf->accel.accel_z;
-        break;
-    case IMU_CHANNEL_YAW:
-        ret = read_buf->gyro.gyro_z;
-        break;
-    case IMU_CHANNEL_PITCH:
-        ret = read_buf->gyro.gyro_x;
-        break;
-    case IMU_CHANNEL_ROLL:
-        ret = read_buf->gyro.gyro_y;
-        break;
-    default:
-        break;
-
-    }
-
-    return ret;
+        /*
+         * Mapped to SAE J670E standard. Front of unit is RJ-45 and USB
+         * plug facing rear of vehicle when mounted on windshield.
+         * Issue #425
```

Comment:
Actually, to match the IMU's natural mounting orientation of MK2 it should match MK2.

MK2:  mounted flat with green terminal block facing forward
RCT: mounted flat with USB / RJ45 facing forward